compiler/damlc/tests/daml-test-files/List.daml

```diff
@@ -325,12 +324,6 @@ testFoldr1 = scenario do
   "abc" === foldr1 (<>) ["a", "b", "c"]
   2 === foldr1 (-) [1, 2, 3]
 
-testFoldable = scenario do
-  "abc" === Foldable.foldl1 (<>) ["a", "b", "c"]
-  -4 === Foldable.foldl1 (-) [1, 2, 3]
-  "abc" === Foldable.foldr1 (<>) ["a", "b", "c"]
-  2 === Foldable.foldr1 (-) [1, 2, 3]
```

Comment:
This does not test any of default implementation for `foldMap`, `foldl` nor `folr`